mockserver/mockserver.go

```diff
@@ -89,6 +100,43 @@ func (h *MockServerHandler) GetBlockHeader(blockHash *chainhash.Hash) (*BlockHea
 // 	return in
 // }
 
+func (h *MockServerHandler) GetTxOut(
+	txHash *chainhash.Hash,
+	index uint32,
+	mempool bool,
```

Comment:
Why do we need mempool arg?

Reply:
It's to keep it compliant with bitcoin RPC spec.